src/views/htmlcontent/src/js/utils.ts

```diff
@@ -58,8 +58,9 @@ export function parseNumAsTimeString(value: number): string {
     let hs = h < 10 ? '0' + h : '' + h;
     let ms = m < 10 ? '0' + m : '' + m;
     let ss = s < 10 ? '0' + s : '' + s;
+    let mss = tempVal < 10 ? '00' + tempVal : tempVal < 100 ? '0' + tempVal : '' + tempVal;
```

Comment:
Suggestion: Couldn't you just check the length of the mss value and if < 3 left pad with 0s? Similar behavior for hrs and seconds too - have a function that takes an int, required #digits (2 or 3) and pads as needed